lib/containers/podman.pm

```diff
@@ -30,4 +30,12 @@ sub configure_insecure_registries {
     assert_script_run('echo -e \'[[registry]]\nlocation = "' . registry_url() . '"\ninsecure = true\' >> /etc/containers/registries.conf') if get_var('REGISTRY');
 }
 
+sub get_storage_driver {
+    my $json = shift->info(json => 1);
+    my $storage = $json->{store}->{graphDriverName};
+    record_info 'Storage', "Detected storage driver=$storage";
```

Comment:
Same as above, isn't a `record_info` directly in a library function a bit too verbose?

Reply:
I still don't agree, but it's not hurting us so let's just move on